temporian/core/operators/all_operators.py

```diff
@@ -19,6 +18,8 @@
 from temporian.core.operators.arithmetic_scalar import subtract_scalar
 from temporian.core.operators.arithmetic_scalar import negate
 from temporian.core.operators.arithmetic_scalar import equal_scalar
+from temporian.core.operators.arithmetic_scalar import greater_scalar
+from temporian.core.operators.arithmetic_scalar import less_scalar

```

Comment:
s/less/lesser in both less and less_scalar, for consistency with "greater"

in case lesser sounds weird, naming them gt and lt could work too? but I think I prefer lesser

Reply:
It seems both numpy and tensorflow use less/greater. I would keep it this way for consistency with this libraries :)
(also discussed above).
(*Added todo on whether we want to change it)